skills/dff_voice_skill/scenario/response.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import common.dff.integration.context as int_ctx
+
+from df_engine.core import Context, Actor
+
+
+logger = logging.getLogger(__name__)
+
+
+def caption(ctx: Context, actor: Actor, excluded_skills=None, *args, **kwargs) -> str:
+    cap = "ERROR"
+    if not ctx.validation:
```

Comment:
if you use get_last_human_utterance method, you do not need this check

Reply:
when you use int_ctx.get_last_human_utterance(ctx, actor) and methods get (as below) you will not face validation problems

Reply:
removed